media/src/components/chat/UserAvatar.tsx

```diff
@@ -0,0 +1,20 @@
+import { Avatar, AvatarBadge, Tooltip } from "@chakra-ui/react";
+import { faker } from "@faker-js/faker";
+
+type props = {
+    name: string;
+}
+
+const UserAvatar = ({name}: props) => {
+    return (
+        <Tooltip label={name}>
+            <Avatar name={name} src={faker.image.avatar()}>
```

Comment:
by calling `faker` here it pulls away from the pureness of this component. this state should be generated from where ever this component is rendered from and passed in as a prop.